packages/inquirer-common/src/types.ts

```diff
@@ -53,6 +53,10 @@ export interface GuiOptions {
      * Indicate state in the left hand navigation panel in YUI
      */
     breadcrumb?: boolean | string;
+    /**
+     * Indicates the type of the prompt
+     */
+    type?: string;
```

Comment:
Can you please add this as a restricted template literal type?

Reply:
Fixed with: e192056901a826ecd83af9c45188ea1c68231240